### PR TITLE
Remove some no-parallel tags from tests (Part 3)

### PR DESCRIPTION
**Pull request by Algunenano**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Remove some no-parallel tags from tests (Part 3)

Continues from https://github.com/ClickHouse/ClickHouse/pull/67673

### Documentation entry for user-facing changes

- [ ] Documentation is written (mandatory for new features)

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

#### CI Settings (Only check the boxes if you know what you are doing):
- [ ]  Allow: All Required Checks
- [ ]  Allow: Stateless tests
- [ ]  Allow: Stateful tests
- [ ]  Allow: Integration Tests
- [ ]  Allow: Performance tests
- [ ]  Allow: All Builds
- [ ]  Allow: batch 1, 2 for multi-batch jobs
- [ ]  Allow: batch 3, 4, 5, 6 for multi-batch jobs
---
- [ ]  Exclude: Style check
- [ ]  Exclude: Fast test
- [ ]  Exclude: All with ASAN
- [ ]  Exclude: All with TSAN, MSAN, UBSAN, Coverage
- [ ]  Exclude: All with aarch64, release, debug
---
- [ ]  Do not test
- [ ]  Woolen Wolfdog
- [ ]  Upload binaries for special builds
- [ ]  Disable merge-commit
- [ ]  Disable CI cache

### CODE REVIEW

**Comment by robot-ch-test-poll** [comment]
*This is an automated comment for commit 53d13f657b087388e3c229d348eac6730de66cc7 with description of existing statuses. It's updated for the latest CI running*

[✅ Click here](https://s3.amazonaws.com/clickhouse-test-reports/67914/53d13f657b087388e3c229d348eac6730de66cc7/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Builds</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/67923/f2591bd1a6401337de0e3bf8d5c1fa93fd1b9394/builds/report.html">✅ success</a></td></tr>
<tr><td>Unit tests</td><td>Runs the unit tests for different release types</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/feeb945461951de1e252753574492a00145355d1/unit_tests__release_.html">✅ success</a></td></tr>
<tbody>
</table>
</details>

File: tests/queries/0_stateless/01056_window_view_proc_hop_watch.py [hunk 1]
```diff
@@ -1,5 +1,4 @@
 #!/usr/bin/env python3
-# Tags: no-parallel
```
Comment (rschu1ze) [inline tests/queries/0_stateless/01056_window_view_proc_hop_watch.py:2]: (I did actually not know that clickhouse-test can run Python tests 😄)

**Review by rschu1ze — APPROVED** [review]

File: tests/queries/0_stateless/01056_window_view_proc_hop_watch.py (hunk 1 above)
Comment (Algunenano) [reply tests/queries/0_stateless/01056_window_view_proc_hop_watch.py:2]: I would be happier if it couldn't :smile: They are harder to run and harder to debug

**Comment by Algunenano** [comment]
Looking into the issue from flaky check for `01056_window_view_proc_hop_watch`. It seems the `WATCH` client did not get the inserted row

**Comment by Algunenano** [comment]
I'm going to remove the changes for 01056_window_view_proc_hop_watch since I'm fixing the feature itself in https://github.com/ClickHouse/ClickHouse/pull/67983. Later we can reintroduce them to make it parallelizable

**Pull request merged**